pkg/visitors/insert.go

```diff
@@ -186,4 +222,12 @@ func init() {
 		"insert a file before another file", 2, genInsertCLI(InsertBefore))
 	RegisterCLI(insertTypeNames[ReplaceFFS],
 		"replace a file with another file", 2, genInsertCLI(ReplaceFFS))
+	RegisterCLI(insertTypeNames[InsertPadFront],
```

Comment:
Instead of creating all these types and commands, perhaps we should rethink this and take an extra "positional" argument in the CLI, and then configure the visitor based on that! the position could be an enum that we extend. I understand if you want to just get this in first tho! we can file an issue to consolidate the insert CLI later.

Reply:
Good point. Will do.

Reply:
Fixed.